src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -39,11 +39,22 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _actualFont{ DEFAULT_FONT_FACE.c_str(), 0, 10, { 0, DEFAULT_FONT_SIZE }, CP_UTF8, false },
         _touchAnchor{ std::nullopt },
         _leadingSurrogate{},
-        _cursorTimer{}
+        _cursorTimer{},
+        _tsfInputControl{ nullptr }
     {
         _Create();
     }
 
+    inline winrt::Windows::UI::Color ColorRefToColor(const COLORREF& colorref)

```

Comment:
This is duplicated in TSFInputControl.cpp, this needs to be in a better spot, any suggestions?

Reply:
Normally we'd put stuff like this into the "types" library but I don't want to pollute that with WinRT things since they blow up the pchs.

We almost need like a `TerminalUtils` lib and a `UnitTests_TerminalUtils` lib...

Reply:
This is the last thing in the Draft PR to resolve.  I don't see anything like this since I started the work.  Any new thoughts on where might be a good idea to place this?

Reply:
Check out `WinRTUtils`

Reply:
I'm moving this to WinRTUtils, will be in next commit.